ocs_samples/basic_samples/Authentication/ClientCredentialFlow/DotNet/ClientCredentialFlow/Program.cs

```diff
@@ -11,33 +11,57 @@ public static class Program
         public static void Main(string[] args)
         {
             InitConfig();
-                
+
             ClientFlow.OcsUrl = GetConfigValue("Resource");
 
             var tenantId = GetConfigValue("TenantId");
             var clientId = GetConfigValue("ClientId");
             var clientSecret = GetConfigValue("ClientKey");
+            var version = GetConfigValue("ApiVersion");
             ClientFlow.CreateAuthenticatedHttpClient(clientId, clientSecret);
 
             // Make an HTTP request to OCS using the authenticated client - since this is the first request, the AuthenticationHandler will
             // authenticate and acquire an Access Token and cache it.
-            var response = ClientFlow.AuthenticatedHttpClient.GetAsync($"api/Tenants/{tenantId}").Result;
-            response.EnsureSuccessStatusCode();
-            Console.WriteLine(response.Content.ReadAsStringAsync());
-            Console.WriteLine($"HTTP GET api/Tenants/{tenantId} successful");
+            try
+            {
+                var response = ClientFlow.AuthenticatedHttpClient.GetAsync($"api/{version}/Tenants/{tenantId}/Users").Result;
+                response.EnsureSuccessStatusCode();
+                Console.WriteLine(response.Content.ReadAsStringAsync());
```

Comment:
This writes: System.Threading.Tasks.Task`1[System.String].  Should we add a .Result to get the string instead of the task?